### DISCUSSION TITLE
Specify Climbing Rate in Dynamic Way

### DISCUSSION THREAD

**Discussion by hjorthchristian** [JSBSim-Team/jsbsim#967] [Q&A]
Hello I was trying to know if it is possible to specify the climbing rate in a more dynamic way, for example for the c172 to fly from one waypoint to another witha certain clibing rate for example 10 ft/s and for another waypoint with 20 ft/s. Is it possible in the version of JSBSim as it is? Or is it necessary to make a custom version and play with the source code. BTW I have seen that there are defined climbing rates in terms of altitude but I would llike to see if it is possible to have more control over it. Thanks in advance!

**Comment by seanmcleod** [comment]
There is no need to modify the JSBSim source code to implement this sort of control. You could implement an autopilot which had setpoints for the climb rate and implement your autopilot along these lines, i.e. is an xml based system external to the JSBSim source code.

https://github.com/JSBSim-Team/jsbsim/blob/master/systems/Autopilot.xml
